charters/DataPortal/charter.md

```diff
@@ -0,0 +1,81 @@
+* Coordinate development of developer documentation content (e.g. “Develop”) from multiple contributors. Includes API documentation and development guides.
+* Click through to Data Browser from search box and human body image
+* Registration form for tertiary portals and list of portals available with click through to hosting web site (e.g. Dockstore)
+* Integration with Zendesk help desk
+* Integration with Vanilla Forums (discussion forum for developers)
+* Integration with Canny (to capture user feedback)
+### Data Browser
+* Create (or use Data Storage Service [DSS] provided) indexes to access files in the Data Store by using public DSS APIs to read the metadata
+* Browser support for multiple metadata schemas that are supported by the schema access library maintained by the Ingest service
+* Search views of Specimens, Projects and Files
+* Browser UI to search the DSS by specifying multiple search criteria (“facets”)
+* Download selected files from the DSS via the UI and/or HCA DCP Command Line Interface (CLI)
+* Leverage the authentication and authorization implementation from the Data Store
+* Browser interface to the DSS Collections service to save, view, list, share and delete the query results from a search operation
+* Handoff of a Collection of search results to a tertiary portal. Provide for direct handoff of search results to FireCloud.
+* Facilities to save, view, share, update, and delete search queries and personal collections on a per-user basis
+* Search results handoff to the Matrix service
+* API Documentation for portable, cloud-neutral APIs
+
+## Out-of-scope
+* Production operation of other HCA DCP components in AWS (e.g. the DSS, data ingestion process, Secondary Analysis workflows, Tertiary Analysis portals, etc.)
+* Any separate data ingest application developed by The European Bioinformatics Institute (EBI) which may replace or augment the "Contribute" tab in the Data Portal
+* Methods Registry providing management, searching, filtering, sorting, visualization, and methods packages in the tertiary portal (registry is developed and maintained by the Broad Institute)
+* The HCA DCP CLI infrastructure implementation (which is part of the DSS)
+* Primary interface for collecting user feedback and feature/bug requests
+
+## Milestones
+* Oct 2018:  Deploy MVP feature set as part of HCA DCP Pilot phase
+* Nov 2018:  Present Pilot phase portal to community
+* EOY 2018:  Improved scaling/hardening for additional gaps identified in HCA DCP Pilot phase
+* 1H  2019:  Save search queries and query results (using DSS Collections APIs). Handoff of a Collection of search results to the Matrix service and to tertiary portals. Vanilla Forums support. Canny support.
+
+## Roles
+
+### Project Lead
+[Brian O’Connor](mailto:brocono@ucsc.edu) 
+
+### Product Owner
+[Trevor Heathorn](mailto:theathor@ucsc.edu) 
+
+### Technical Lead
+Portal: [Dave Rogers](mailto:dave@clevercanary.com)
+Browser: [Hannes Schmidt](mailto:hannes@ucsc.edu)
+
```

Comment:
Do you want to mention the overall dev team size this charter is scoped for?  That was something we discussed in the UCSC arch meeting today.

Reply:
Related to this... is there a place in the charter that references the time period you think the charter will be valid for?  Are we going for, say, a charter that should be generally accurate for a year and then plan to revise the charter yearly?

Reply:
@briandoconnor - Both questions were answered inline - https://github.com/HumanCellAtlas/dcp-community/pulls#issuecomment-412667983